resources/lib/objects/kodi/music.py

```diff
@@ -189,14 +189,15 @@ def add_genres(self, kodi_id, genres, media):
 
         ''' Add genres, but delete current genres first.
         '''
+		#Removed for DB version 72
```

Comment:
can you use consistent whitespace? here are tabs while the file uses spaces.

Reply:
Will re-submit in the future Leia branch ;)